README.md

```diff
@@ -22,10 +22,12 @@ Thanks to the team behind **twitter-together** for putting together a wonderful
 
 ## Posting a new tweet
 
-1. Fork this repository
-2. Create a new `.tweet` file with the tweet contents. Make sure the file is as per the [guidelines](GUIDELINES.md)
-3. Create a PR to this repository
-4. Once the PR is approved and merged, the tweet is sent out from the [K8sContributors account](https://twitter.com/k8scontributors)
+1. Create a [new issue](https://github.com/kubernetes-sigs/contributor-tweets/issues/new/choose) in this reopository.
```

Comment:
It would be nice to provide two workflows for the tweets here. One through the 'fork and PR' way and the other through an 'issue'. This way we can let users choose the one they feel comfortable with. What do you think?